examples/bench_mark/Makefile

```diff
@@ -0,0 +1,5 @@
+build:
+	docker build -t gcr.io/insprlabs/inspr/example/sender:latest -f sender/sender.Dockerfile ../..
```

Comment:
Just a suggestion: put the sender and receiver in a more specific folder. Instead of "example", use something like "send-receiver-example" ou "throuput-app".